content/spin/v2/python-components.md

```diff
@@ -74,12 +84,67 @@ A new Python component can be created using the following command:
 $ spin new -t http-py hello-world --accept-defaults
 ```
 
-This creates a directory of the following structure:
+### System Housekeeping (Use a Virtual Environment)
+
+Once the component is created, we can change into the `hello-world` directory, create and activate a virtual environment and then install the component's requirements:
+
+<!-- @selectiveCpy -->
+
+```console
+$ cd hello-world
+```
+
+Install the `virtualenv` tool:
+
+<!-- @selectiveCpy -->
+
+```console
+$ pip install virtualenv
+```
+
+Create a virtual environment directory (we are still inside the Spin app directory):
+
+<!-- @selectiveCpy -->
+
+```console
+# python<version> -m venv <virtual-environment-name>
+$ python3.12 -m venv venv-dir
```

Comment:
```suggestion
$ python3 -m venv venv-dir
```
I don't think we need to be quite so specific about which Python to use -- any recent 3.x release should work fine.